great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -1081,7 +1099,7 @@ def as_selectable(self) -> sqlalchemy.Selectable:
         return sa.table(self.table_name, schema=self.schema_name)
 
     @override
-    def _create_batch_spec_kwargs(self) -> dict[str, Any]:
+    def _create_batch_spec_kwargs(self) -> Dict[str, Any]:
```

Comment:
Pydantic picked a very unfortunate name for the serialization method in v1. Getting around this by using `typing.Dict` instead of `dict`